Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,55 @@
+### Note: This DockerFile is adapted from https://github.com/ucb-bar/chipyard/blob/master/.circleci/images/Dockerfile which was adapted from: https://github.com/CircleCI-Public/example-images/openjdk
+# This is a full chipyard setup
+
+FROM ubuntu:18.04
+ARG CHIPYARD_HASH
```

Comment:
Is there any way to specify a "default" argument?

Reply:
There is a way, but it also just stays on the most recent master branch commit by default even when no argument is given. Would we want the default to be a different commit?

Reply:
Actually, I think the main Dockerfile (the one with the toolchains) doesn't need a default argument. If an external user tries to use the Dockerfile (modify and build it themselves) I think pointing them to the most recent Chipyard is fine. 

As for the CI Dockerfile, I think knowing what commit was used is valuable (unless we somehow bake in that information into the version - which is another option).